packages/react-mutation-mapper/src/component/column/Hgvsg.tsx

```diff
@@ -0,0 +1,97 @@
+import { DefaultTooltip } from 'cbioportal-frontend-commons';
+import { generateHgvsgByMutation, Mutation } from 'cbioportal-utils';
+import { observer } from 'mobx-react';
+import * as React from 'react';
+
+import { defaultSortMethod } from '../../util/ReactTableUtils';
+
+import hgvsgStyles from './hgvsg.module.scss';
+
+type HgvsgProps = {
+    mutation: Mutation;
+    constructHref?: (hgvsg: string) => string;
+    constructLink?: (
+        hgvsg: string,
+        content: JSX.Element,
+        href?: string
+    ) => JSX.Element;
+    mainContent?: (hgvsg?: string) => JSX.Element;
+    tooltipContent?: (hgvsg?: string) => JSX.Element;
+    disableTooltip?: boolean;
+};
+
+export function hgvsgSortMethod(a: string, b: string) {
+    return defaultSortMethod(a, b);
+}
+
+@observer
+export default class Hgvsg extends React.Component<HgvsgProps, {}> {
+    public static defaultProps: Partial<HgvsgProps> = {
+        constructLink: (hgvsg: string, content: JSX.Element, href?: string) => (
+            <a
+                href={href || `https:/www.genomenexus.org/variant/${hgvsg}`}
+                target="_blank"
+                rel="noopener noreferrer"
+            >
+                {content}
+            </a>
+        ),
+        mainContent: (hgvsg?: string) => (
+            <>
+                {hgvsg}&nbsp;
+                <i className="fa fa-external-link" />
```

Comment:
Will it be too much work to make this link out icon optional? I think we need it in cbioportal mutation table since it links to genome nexus, but in signal we are just directing to variant page inside signal. If too much effort then let's keep it as what it is

Reply:
Makes sense. I don't think it would need too much effort.

Reply:
Actually, we can just pass a `mainContent` prop in signal to remove the link out icon. I think it is okay to have Genome Nexus as a default link out. Signal is the specific use case, so let's make the change there.